### PR TITLE
669

### LINKED ISSUE #669: Files ignored on push directly after clone
## Expected Behavior
After cloning a script, any subfolder files (e.g. gs/Code.gs) are ignored when subsequently running `clasp status` and ignored and deleted from the script file when running `clasp push`.

## Actual Behavior
Subfolder files are ignored.
```
user@localhost:~/Downloads/clasp/clasptest$ sudo clasp clone "xxxxx"
Cloned 3 files.
└─ gs/inner.js
└─ appsscript.json
└─ Code.js
user@localhost:~/Downloads/clasp/clasptest$ sudo clasp status
Not ignored files:
└─ Code.js
└─ appsscript.json

Ignored files:
└─ .clasp.json
└─ gs/inner.js
```

## Steps to Reproduce the Problem

1. Clone script
2. Run `clasp status` or `clasp push` = Subfolder files ignored

## Specifications

- Node version (`node -v`): 8.16.0
- Version (`clasp -v`): 2.2.1
- OS (Mac/Linux/Windows): Ubuntu 16.04 running on ChromeOS Crouton

### PR DESCRIPTION
**Pull request by PopGoesTheWza**
Fixes #669 

- [x] `npm run test` succeeds.
- [x] `npm run lint` succeeds.
- [ ] Appropriate changes to README are included in PR.

### CODE REVIEW

**Comment by PopGoesTheWza** [comment]
@grant `clasp create` tests fails because of quota issue I guess. Will Travis automatically rerun later (hopefully after quota refresh)

**Comment by grant** [comment]
Thanks.
Please don't have so many commits in your PRs or else you'll exhaust the quota. Just have 1 commit by pulling clasp's `master` first, then creating a branch and PR.

**Comment by PopGoesTheWza** [comment]
> Just have 1 commit by pulling clasp's `master` first, then creating a branch and PR.

It must be me doing something wrong. I'll try to find why old commits keep showing up in PR.

**Comment by PopGoesTheWza** [comment]
@grant what do you think of this PR?

File: src/files.ts [hunk 1]
```diff
@@ -348,11 +345,11 @@ export async function pushFiles(silent = false) {
...
-          console.log(`└─ ${filePath}`);
-        });
+        console.log(listFilesHelper(nonIgnoredFilePaths));
         console.log(LOG.PUSH_SUCCESS(nonIgnoredFilePaths.length));
       }
     }
   });
 }
+
+export const listFilesHelper = (files: string[]) => files.map(file => `└─ ${file}`).join(`\n`);
```
Comment (grant) [inline src/files.ts]: - Change `listFilesHelper` to `logFileList`
- `console.log` the fileMapString in this method.

**Comment by grant** [comment]
Thanks!

**Pull request merged**